src/config.rs

```diff
@@ -115,6 +115,12 @@ impl Environment {
             .ok()
     }
 
+    fn ignore_cargo_config(&self) -> Option<bool> {
+        env::var("CROSS_IGNORE_CARGO_CONFIG")
```

Comment:
```suggestion
        self.get_var("CROSS_IGNORE_CARGO_CONFIG")
```

Reply:
Should we support this in `Cross.toml` actually?